web/templates/gear_menu_popover.hbs

```diff
@@ -0,0 +1,174 @@
+<ul class="navbar-dropdown-menu" id="gear-menu-dropdown" aria-labelledby="settings-dropdown" data-simplebar>
+    <div class="org-info">
+        <li class="org-name">{{realm_name}}</li>
+        <li class="org-url">{{realm_url}}</li>
+        {{#if is_self_hosted }}
+            <li class="org-version">
+                <a href="#about-zulip" class="navigate-link-on-enter">{{version_display_string}}</a>
+            </li>
+            {{#if server_needs_upgrade }}
+            <li class="org-upgrade">
+                <a href="https://zulip.readthedocs.io/en/stable/production/upgrade.html" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t 'Upgrade to the latest release' }}</a>
+            </li>
+            {{/if}}
+        {{else}}
+            <li class="org-plan hidden-for-spectators">
+                {{#if is_plan_limited }}
+                    <a href="/plans/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">Zulip Cloud Free</a>
+                {{else if is_plan_standard}}
+                    <a href="/plans/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">Zulip Cloud Standard</a>
+                {{else if is_plan_standard_sponsored_for_free}}
+                    <a href="/plans/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">Zulip Cloud Standard (sponsored)</a>
+                {{/if}}
+            </li>
+        {{/if}}
+        {{#if (and is_plan_limited is_owner) }}
+        <li class="org-upgrade">
+            <a href="/upgrade/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t "Upgrade to {standard_plan_name}" }}</a>
+        </li>
+        {{/if}}
+        {{#if is_plan_limited }}
+            {{#if is_education_org }}
+            <li class="org-upgrade">
+                <a href="/upgrade/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t 'Request education pricing' }}</a>
+            </li>
+            {{else if (not is_business_org) }}
+            <li class="org-upgrade">
+                <a href="/upgrade/" target="_blank" rel="noopener noreferrer" class="navigate-link-on-enter">{{t 'Request sponsorship' }}</a>
+            </li>
+            {{/if}}
+        {{/if}}
+    </div>
+    <div class="navbar-dropdown-divider" role="presentation"></div>
+    <li class="link-item hidden-for-spectators" role="presentation">
+        <a href="#streams/subscribed" class="navigate-link-on-enter">
+            <i class="zulip-icon zulip-icon-hash" aria-hidden="true"></i> {{t 'Manage streams' }}
+        </a>
+    </li>
+    <li class="link-item hidden-for-spectators" role="presentation">
+        <a href="#settings" class="navigate-link-on-enter">
+            <i class="zulip-icon zulip-icon-tool" aria-hidden="true"></i> {{t 'Personal settings' }}
```

Comment:
All the new icons added and referenced by this PR need to be documented in `docs/THIRDPARTY`

Reply:
Looks like don't have that icons in personal menu too. @terpimost would you say is the source of the icons used for gear menu and personal menu?

Reply:
Updated after talking with Vlad.